src/main/java/frc/robot/subsystems/ClawSubsystem/ClawSubsystem.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems.ClawSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+public class ClawSubsystem extends SubsystemBase{
+    private TalonSRX motor1;
+    private Solenoid pfft1;
+
+    public ClawSubsystem() {
+        
+      CommandScheduler.getInstance().registerSubsystem(this);
+
+        motor1 = new TalonSRX(0);
+        pfft1 = new Solenoid(1);
+    }
+    
+    @Override
+    public void periodic() {
+      // This method will be called once per scheduler run
+
+    }
+  
+    @Override
+    public void simulationPeriodic() {
+      // This method will be called once per scheduler run during simulation
+    }
+
+    public void openClaw(){
+        motor1.set(ControlMode.PercentOutput, .5);
```

Comment:
I think that we should stick to our state system, in that we should have states that are public field variable (e.g. "public boolean openClaw"), and then take action in periodic() based on that field variable. same applies for lift methods.

On top of that, we should probably use encoders and run the motor until it stops moving (rate of change of encoder reading is basically 0), rather than keeping it powered. We have a high chance of burning out motors if we just set power.